common/app/routes/Challenges/views/step/Show.jsx

```diff
@@ -27,9 +31,12 @@ const renderPane = name => {
   return Comp ? <Comp /> : <span>Pane { name } not found</span>;
 };
 
-export default function ShowStep() {
+export default function ShowStep({ showLoading }) {
   return (
     <ChildContainer isFullWidth={ true }>
+      {
+        showLoading ? <OverlayLoader /> : null
+      }
```

Comment:
There is a repeated pattern here. We can make this DRY by moving this up into the Challenges/Child-Contianer.jsx